oak/server/oak_runtime.cc

```diff
@@ -158,20 +156,6 @@ bool OakRuntime::CreateAndRunNode(const std::string& config_name,
 }
 
 grpc::Status OakRuntime::Start() {
-  // We call into the Rust runtime to verify that bindings between C++ and Rust are working
```

Comment:
Why is this code being deleted?
Are we deprecating the Rust Runtime?

Reply:
No, please see @blaxill's comment below.